presidio-anonymizer/tests/test_analyzer_results.py

```diff
@@ -0,0 +1,40 @@
+import json
+import os
+
+from presidio_anonymizer.entities import AnalyzerResults, AnonymizerRequest
+
+
+def test_analyzer_results_not_failing_on_empty_list():
+    analyzer_result = AnalyzerResults()
+    assert len(analyzer_result.to_sorted_unique_results()) == 0
+
+
+def test_analyzer_results_sorted_set():
+    json_path = json_path = file_path("dup_payload.json")
```

Comment:
consider turning into a fixture. Makes code a little more readable and would only require opening the file once for all tests.

Reply:
Done.